groups/cvo2/security_groups.tf

```diff
@@ -0,0 +1,104 @@
+module "netapp_secondary_security_group" {
+  source  = "terraform-aws-modules/security-group/aws"
+  version = "~> 3.0"
+
+  name        = "sgr-netapp-new-${var.account}-002"
+  description = "Secondary security group for the NetApp CVO Service"
+  vpc_id      = data.aws_vpc.vpc.id
+
+  tags = merge(
+    local.default_tags,
+    map(
+      "Name", "sgr-netapp-new-${var.account}-002",
+      "ServiceTeam", "Storage"
+    )
+  )
+}
+
+resource "aws_network_interface_sg_attachment" "cvo_instance_sgr_attachment" {
+  for_each = toset(data.aws_network_interfaces.netapp.ids)
+
+  security_group_id    = module.netapp_secondary_security_group.this_security_group_id
+  network_interface_id = each.value
+
+  depends_on = [
+    module.netapp_secondary_security_group,
+    data.aws_network_interfaces.netapp
+  ]
+}
```

Comment:
This entire resource should be commented until the initial deployment has been created as the data lookup it depends on should also be commented.